TowerDefenseGame/Assets/Tests/PlayerBaseTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEngine.UI;
+
+namespace Tests
+{
+    public class PlayerBaseTests
+    {
+        private PlayerBase playerBase;
+        private Text healthText;
+
+        [SetUp]
+        public void Setup()
+        {
+            var prefab = AssetDatabase.LoadAssetAtPath("Assets/Prefabs/PlayerBase.prefab", typeof(PlayerBase));
```

Comment:
That's an interesting way to load a prefab. I've always made it a public variable and set it in the Unity inspector.